doc-content/enterprise-only/processes/interacting-with-processes-resuming-tasks-in-bulk-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='interacting-with-processes-resuming-tasks-in-bulk-proc']
+= Resuming tasks in bulk
+
+After tasks are created in {CENTRAL}, you can resume your tasks in bulk.
```

Comment:
What you can do is resume suspended tasks available to you, it's not enough that task are created in {CENTRAL}
Maybe something like this 'After task are suspended in {CENTRAL} they can be resumed in bulk'

Reply:
@nmirasch, Ack